### PR TITLE
New page for the commercial traffic twinning dataset

### PR DESCRIPTION
**Pull request by RavisBlue**

### CODE REVIEW

File: _posts/datasets/2024-09-30-open-ran-commercial-traffic-twinning-dataset.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "Open-RAN Commercial Traffic Twinning Dataset"
```
Comment (leonardobonati) [inline _posts/datasets/2024-09-30-open-ran-commercial-traffic-twinning-dataset.md:3]:
```suggestion
title: "Open RAN Commercial Traffic Twinning Dataset"
```

File: _posts/datasets/2024-09-30-open-ran-commercial-traffic-twinning-dataset.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+<img src="/assets/post-assets/traffic-twinning-pipeline.png" class="post-image" alt="ColO-RAN Architecture" width="50%">
+
+We emulate a 5G network with 1 base station and 8 users in the dense urban scenario of Rome, Italy. We consider a multi-slice scenario in which the users are statically assigned to a slice of the network. The traffic they request is generated by MGEN and falls into two different traffic types: high-capacity enhanced mobile broadband (eMBB) for UEs 1-4 allocated in slice 0, and ultra-reliable low-latency communications (URLLC) for UEs 5-8 allocated in slice 1. The traffic is twined from real-world traces from base stations in a suburban area of Madrid, covering three different spectrum bands (816 MHz, 1835 MHz, and 2650 MHz). For the same traffic and slicing configutation, we experiment with both schedualing policies proportionally fair (PF) and round-robin (RR). The dataset contains five different slicing configurations, each showing how the 50 PRBs (10 MHz of spectrum) are allocated between the two slices. The details are provided in the following tables.
```
Comment (leonardobonati) [inline _posts/datasets/2024-09-30-open-ran-commercial-traffic-twinning-dataset.md:27]:
```suggestion
We emulate a 5G network with 1 base station and 8 users in the dense urban scenario of Rome, Italy. We consider a multi-slice scenario in which the users are statically assigned to a slice of the network. The traffic they request is generated by MGEN and falls into two different traffic types: high-capacity enhanced Mobile Broadband (eMBB) for UEs 1-4 allocated to slice 0, and Ultra-reliable Low-latency Communications (URLLC) for UEs 5-8 allocated to slice 1. The traffic is twinned from real-world traces from base stations in a suburban area of Madrid, covering three different spectrum bands (816 MHz, 1835 MHz, and 2650 MHz). For the same traffic and slicing configuration, we experiment with the proportionally fair and round-robin scheduling policies. The dataset contains five different slicing configurations, each showing how the 50 PRBs (10 MHz of spectrum) are allocated between the two slices. The details are provided in the following tables.
```

**Review by leonardobonati — CHANGES_REQUESTED** [review]
Thank you for putting this together. Can you add links to the Madrid dataset, and to MGEN?

**Comment by Thecave3** [comment]
Try to use markdown for links!

**Review by leonardobonati — APPROVED** [review]

**Pull request merged**